src/hooks/useNavigation.js

```diff
@@ -20,8 +19,9 @@ export const useNavigation = defaultSection => {
 
 const handleKeyPress = (event, focusables, defaultSection) => {
   const { key, metaKey } = event;
+  const { activeElement } = document;
 
-  if (NON_NAVIGATION_KEYS.includes(key)) return;
+  if (!activeElement.hash && NON_NAVIGATION_KEYS.includes(key)) return;
```

Comment:
Question: If I press SPACE for example, and I'm focused on an element that has a `hash`, we continue here?

I'm curious about the nature of this condition. Can you enlighten me?

Reply:
If we press enter on element then we will return from here, but if clicked element has hash then we don't want to return we want to continue and our further condition will focus the next element as required. Further we only want to focus the hashed element if enter key is pressed,  rest all cases are AFAIK will ignored by all the condition and do not harm our existing behaviour.